BETA.md

```diff
@@ -1,69 +1,96 @@
-of collecting valuable feedback from real-world scenarios.
+The following instructions are intended for contributors who want to try Metals
+on a real-world project.
 
 # Global setup
+
 These steps are required once per machine.
 
 ## Step 1 - sbt plugin
-The server needs to access some metadata about the build configuration. This data are produced by
-an sbt plugin. This plugin is currently not published, so you will need to copy paste it on your machine.
 
-Here's the source of the plugin: https://github.com/scalameta/metals/blob/master/project/MetalsPlugin.scala
+The server needs to access some metadata about the build configuration. This
+data are produced by an sbt plugin. This plugin is currently not published, so
+you will need to copy paste it on your machine.
+
+Here's the source of the plugin:
+https://github.com/scalameta/metals/blob/master/project/MetalsPlugin.scala
 
 Copy the source to either (depending on your sbt version):
-- (sbt 0.13) `~/.sbt/0.13/plugins/MetalsPlugin.scala`
-- (sbt 1.0) `~/.sbt/1.0/plugins/MetalsPlugin.scala`
+
+* (sbt 0.13) `~/.sbt/0.13/plugins/MetalsPlugin.scala`
+* (sbt 1.0) `~/.sbt/1.0/plugins/MetalsPlugin.scala`
 
 ## Step 2 - build the VSCode extension
-The VSCode extension is not yet published on the Marketplace, so you'll need to build it locally.
-For this step, `node` and `npm` are required.
 
-- `cd vscode-extension`
-- `npm install`
-- `npm run build`
-- `code --install-extension vscode-metals-0.0.1.vsix`
+The VSCode extension is not yet published on the Marketplace, so you'll need to
+build it locally. For this step, `node`, `npm` and VS Code are required.
+
+* `cd vscode-extension`
+* `npm install`
+* `npm run build`
+* `code --install-extension vscode-metals-0.0.1.vsix`
+
+Make sure you have the
+["Scala Syntax"](https://marketplace.visualstudio.com/items?itemName=daltonjorge.scala)
+plugin installed in VS Code.
 
 ## Step 3 - publish the server locally
+
 From the repo root run `sbt publishLocal`
 
 # Per-project setup
+
 These steps are required on each project.
 
 ## Step 1 - add semanticdb-scalac compiler plugin to your project
-Some features of the server rely on the artifacts produced by the [`semanticdb-scalac`](http://scalameta.org/tutorial/#semanticdb-scalac) compiler plugin.
-There are two alternative ways to install `semanticdb-scalac`.
 
-The first option is to enable `semanticdb-scalac` permanently for your project in `build.sbt` with:
+Some features like definition/references/hover rely on artifacts produced by a
+compiler plugin called `semanticdb-scalac`. There are two alternative ways to
+install `semanticdb-scalac`.
+
+The first option is to enable `semanticdb-scalac` permanently for your project
+in `build.sbt` with:
 
 ```scala
-libraryDependencies += compilerPlugin("org.scalameta" % "semanticdb-scalac" % "2.1.2" cross CrossVersion.full)
+ addCompilerPlugin("org.scalameta" % "semanticdb-scalac" % "2.1.2" cross CrossVersion.full)
```

Comment:
Can this be updated to the latest 2.*?

Reply:
Good idea, done.

We will soon need to upgrade to 3.x